src/codeExamples/replacingFlexContainer.stories.mdx

```diff
@@ -0,0 +1,142 @@
+But it also required more definitions for each element, so we have decided to use more predefined layout primitives - Inline, Stack, Center, and Cluster. And we added some extra features.
+
+First a short review of the replacement components called layout primitives.
+
+## [Stack](?path=/docs/layout-primitives-stack--playground)
+
+Stack is a flex container that has predefined flex-direction to a column. Apart from the standard flexbox column, it allows you to add the same size gaps between the elements or insert wider gap (splitAt) after a defined element. 
+Property isRecursive allows you to insert another element with multiple children and apply the same rules to it
+
+## [Inline](?path=/docs/layout-primitives-inline--playground)
+
+Inline is a flex container that has predefined flex-direction to a row. You can adjust justify (justifyContent), align (alignItems), gap size, and stretch.
+
+## [Center](?path=/docs/layout-primitives-center--playground)
+
+Center does what you think it does - it aligns its children to the center horizontally and vertically.
+
+## [Cluster](?path=/docs/layout-primitives-cluster--playground)
+
+Cluster component is quite like inline component, it just allows for wrapping if the width is too small. This component is a good option for navigation.
+
+## Examples
+
+Let's take a look at replacement examples
+
+1.
+
+```jsx
+  <FlexContainer justifyContent="flex-start" alignItems="center">
+    <IhsIcon src={ihsLogo} alt="IHS Markit Logo" />
+    <IhsText src={ihsText} alt="IHS Markit" />
+  </FlexContainer>
+```
+
+FlexContainer is by default row, so it should turn to Inline instead.
+Replace it with inline:
+
+```jsx
+  <Inline justify="flex-start" align="center">
+    <IhsIcon src={ihsLogo} alt="IHS Markit Logo" />
+    <IhsText src={ihsText} alt="IHS Markit" />
+  </Inline>
+```
+
+2.
+
+```jsx
+const BrandingFlexContainer = styled(FlexContainer)`
+  width: 100%;
+`;
+const WhiteLabelingLogoContainer = styled(FlexContainer)`
+  background: none;
+  width: ${pxToRem(160)};
+  height: ${pxToRem(90)};
+  border: 1px solid ${getColor('neutral.400')};
+  margin-bottom: ${pxToRem(15)};
+  padding: ${pxToRem(6)};
+  border-color: ${prop('borderColor')};
+  margin-left: ${pxToRem(35)};
+`;
+...
+
+<BrandingFlexContainer justifyContent="flex-start">
+  <FlexContainer flexShrink={1} flexGrow={1} flexDirection="column">
+    {...someLogic}
+  </FlexContainer>
+
+  <WhiteLabelingLogoContainer
+    flexDirection="row"
+    justifyContent="center"
+    alignItems="center"
+    borderColor={
+      brandingOptions.COMPANY_AND_SSC === selectedBrandingType
+        ? 'transparent'
+        : undefined
+    }
+  >
+    {...moreLogic}
+  </WhiteLabelingLogoContainer>
+</BrandingFlexContainer>
+```
+
+replace with Stack, Inline and Center
+
+```jsx
+const WhiteLabelingLogoContainer = styled(Center)`
```

Comment:
Why did you choose `Center` as a replacement? For me, this is confusing because I can't see any clear sign in the original example that would suggest that this box is centered, like `margin: 0 auto`.